Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+#!groovy
+
+// Jenkinsfile for compiling, testing, and packaging the Dashel libraries.
+// Requires CMake plugin from https://github.com/davidjsherman/aseba-jenkins.git in global library.
+
+pipeline {
+	agent label:'' // use any available Jenkins agent
+	stages {
+		stage('Prepare') {
+			steps {
+				sh 'mkdir -p build dist'
+				dir('dashel') {
+					checkout scm
+				}
+				stash includes: 'dashel/**', excludes: '.git', name: 'source'
+			}
+		}
+		stage('Compile') {
+			steps {
+				unstash 'source'
+				CMake([buildType: 'Debug',
+					   sourceDir: '$workDir/dashel',
+					   buildDir: '$workDir/build/dashel',
+					   installDir: '$workDir/dist',
+					   getCmakeArgs: [ '-DBUILD_SHARED_LIBS:BOOL=ON' ]
```

Comment:
`BUILD_SHARED_LIBS` is on by default, so why do we need this? I would like to avoid duplicating information at different places if possible.